docs/IMAGES.md

```diff
@@ -0,0 +1,54 @@
+# Abstruse CI Images
+
+**For running builds Docker Image is required.**
+
+During the process of Abstruse installation, a new Docker Image with name `abstruse_builder` is built. This Image provides Abstruse
+with all of the necessary data to run builds.
+
+Each config file `.abstruse.yml` in your repositor must start with line `image: "name_of_abstruse_image"`, where `"name_of_abstruse_image"` represents
```

Comment:
TYPO: "repository"

Reply:
In fact, there are a few. I'll wait till the PR goes in and do a new one with corrections :)

Reply:
thanks for watching our back Adam, its nice to have a native English speaker in the team.

Reply:
hehe tnx, we're not english native speaker and it shows quickly when you start reading our documentation :).